src/components/App.jsx

```diff
@@ -36,29 +36,30 @@ export class App extends Component {
         <RealTimeQueries doctype="io.cozy.oauth.clients" />
 
         <Main>
-          <Switch>
-            <Route path="/redirect" component={IntentRedirect} />
-            <Route exact path="/profile" component={Profile} />
-            <Route path="/profile/password" component={Passphrase} />
-            <Route exact path="/connectedDevices" component={Devices} />
-            <Route path="/connectedDevices/:deviceId" component={Devices} />
-            <Route path="/sessions" component={Sessions} />
-            <Route path="/storage" component={Storage} />
-            <Route exact path="/permissions/:page" component={PermissionsTab} />
+          <Routes>
+            <Route path="/redirect" element={<IntentRedirect />} />
+            <Route path="/profile/*" element={<Profile />} />
```

Comment:
are you sure about replacing `exact` by `/*` ? it looks weird

Reply:
Yes, `exact` doesn't exist anymore in v6, it's not even deprecated but removed, they went with some kind of OS-based path patterns. I must say it's a bit confusing at first, and I didn't fully grasp the reasoning behind that change yet

Reply:
thanks for explanation